### DISCUSSION TITLE
Brave and Duckduckgo have already implemented De-AMP. Would Bromite implement that?

### DISCUSSION THREAD

**Discussion by pixincreate** [bromite/bromite#2008] [General]
As of today, Brave and Duckduckgo have implemented a new feature by putting another nail on the Google's coffin called De-AMP (Accelerated Mobile Pages, I'm not sure what's it called btw). 

Bromite is known for it's privacy. Will Bromite implement the same? Would a feature on this topic feasible?

**Comment by mpaunovic** [comment]
With Google proposing a new standard called WebBundles this feature should be taken into a serious consideration.

https://web.dev/web-bundles

https://brave.com/web-standards-at-brave/3-web-bundles/

**Comment by csagan5** [comment]
Bromite has had anti-AMP functionality for years, although it is not 100% working.

> Will Bromite implement the same?

"Bromite" will not implement anything on its own; patches are welcome.

**Reply by pixincreate** [reply]
Yea, I do agree with that. However, Brave have their patches made available on their GitHub page.
